tests/unit/lms/services/blackboard_api/_basic_test.py

```diff
@@ -178,5 +140,4 @@ def basic_client(http_service, oauth_http_service):
         redirect_uri=sentinel.redirect_uri,
         http_service=http_service,
         oauth_http_service=oauth_http_service,
-        refresh_enabled=False,
     )
```

Comment:
Now that the tests are significantly shorter I wonder if we can move to the more standard `Test<class>.test_<method>` layout rather than `Test<method>.test_it` layout like the Canvas API tests?

I suspect it would make the tests a bit shorter and easier to look through. At the moment there are a lot of tests like `test_it_raises_ExternalRequestError_if_the_request_fails` which require you to scroll up to the class name to work out what "it" is in this context.

Reply:
Done